table/options/options.go

```diff
@@ -608,33 +623,76 @@ type (
 	}
 )
 
+// WithAddColumn adds column in AlterTable request
+//
+// Deprecated: use AddColumn instead
```

Comment:
Why you want remove prefix With from some of functions?

other functions exists with the prefix, for example: With Alter Partition Settings Object, With Alter Key Bloom Filter, ... and other options in other packages.

I think prefix removing should be:
1. Separate PR with rename/deprecate only
2. Rename all public options
3. Think about complete remove prefix, or change it, for example: Alter Table Add Column, Alter Table Set TTL, ProfileReplicationPolicy, CreateTableBloomFilter, etc - for uniform naming and prevent names clash.